packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyFilesControls.tsx

```diff
@@ -4,25 +4,26 @@ import { ViewElement } from '../../context/elements';
 
 import { Controls } from '../Controls';
 
-import { Title } from './Controls/Title';
 import { displayText } from '../../displayText/en';
 import { CLASS_BASE } from '../constants';
 import { DestinationPicker } from './DestinationPicker';
 
 import { useCopyView } from './CopyView/useCopyView';
-import { DataTableControl } from '../../controls/DataTableControl';
 import { ControlsContextProvider } from '../../controls/context';
-import { getActionViewTableData } from './utils';
+import { getActionViewTableData, GetTitle } from './utils';
```

Comment:
Why is this function capitalized? Can we please change this back to camelCase?

Reply:
Oops, this was eslint one. Added use to deactivate it